mkdocs.yml

```
@@ -0,0 +1,63 @@
+    - navigation.top
+    - navigation.tracking
+    - search.highlight
+    - search.share
+    - search.suggest
+    - toc.follow
+extra_css:
+  - stylesheets/extra.css
+nav:
+  - Home: "index.md"
+  - Guide:
+    - "Switch Workspace": "guides/switch_workspace.md"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)